db/migrate/20200324104023_add_review_type_to_consultation.rb

```diff
@@ -0,0 +1,5 @@
+class AddReviewTypeToConsultation < ActiveRecord::Migration[6.0]
+  def change
+    add_column :consultations, :review_type, :integer, default: 0
```

Comment:
@BALAJI-BALA - In this migration, also add a script to change the current consultations to review type of `consultation`